src/main/java/com/health/healther/controller/CouponController.java

```diff
@@ -0,0 +1,27 @@
+package com.health.healther.controller;
+
+import javax.validation.Valid;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.health.healther.dto.coupon.CouponCreateDto;
+import com.health.healther.service.CouponService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/coupon")
+public class CouponController {
+
+	private final CouponService couponService;
+
+	@PostMapping
+	public Long addCoupon(@RequestBody @Valid CouponCreateDto createDto) {
+		return couponService.addCoupon(createDto);
+	}
```

Comment:
1. 반환값과 HttpStatus를 담기 위해 ResponseEntity<> 형태로 감싸는게 좋아보이는데 어떻게 생각하시나요? 
```suggestion
	@PostMapping
	public ResponseEntity<Long> addCoupon(@RequestBody @Valid CouponCreateDto createDto) {
		return ResponseEntity.ok(couponService.addCoupon(createDto));
	}
```

Reply:
> 에러 코드와 같은 HTTP상태 코드를 전송하고 싶은 데이터와 함께 전송할 수 있기 때문에 좀 더 세밀한 제어가 필요한 경우 사용한다고 한다.

ResponseEntity 사용하는 이유 확실하게 알고 가게 되었습니다. 감사합니다!